test/xdotoolify-spec.js

```diff
@@ -409,55 +409,51 @@ describe('xdotoolify', function() {
   });
 
   it('should require check after addRequireCheckImmediatelyAfter', async function() {
-    let errorMsg = 'Nothing thrown';
     let goodFunc = Xdotoolify.setupWithPage((page) => { return 5; });
     let fnWithRequire = Xdotoolify.setupWithPage(
       async (page) => {
         await page.X
             .addRequireCheckImmediatelyAfter().do();
       }
     );
-    const noop = () => {};
-
-    try {
-      await page.X
-          .run(goodFunc)
-          .do();
-    } catch (e) {
-      errorMsg = e.message;
-    }
 
-    expect(errorMsg).toBe('Nothing thrown');
+    await page.X
+        .run(goodFunc)
+        .do({legacyCheckUntil: false});
 
-    try {
+    expect(async () => {
       await page.X
           .run(goodFunc)
           .run(fnWithRequire)
-          .do();
-    } catch (e) {
-      errorMsg = e.message;
-    }
-
-    expect(errorMsg).toBe(
-      'Missing checkUntil after running ' +
-      '\'requireCheckImmediatelyAfter\'.'
-    );
-
-    errorMsg = null;
+          .do({legacyCheckUntil: false});
+    }).rejects.toThrow();
 
-    try {
+    expect(async () => {
       await page.X
           .run(goodFunc)
           .addRequireCheckImmediatelyAfter()
           .do();
-    } catch (e) {
-      errorMsg = e.message;
-    }
+    }).rejects.toThrow();
```

Comment:
For future commits, you can use `toThrow('Missing checkUntil after running \'requireCheckImmeidatelyAfter\'.')`

Reply:
Changed.